packages/@react-aria/interactions/test/usePress.test.js

```diff
@@ -591,6 +591,68 @@ describe('usePress', function () {
       ]);
     });
 
+    it('should not ignore virtual pointer events on android ', function () {
+      let uaMock = jest.spyOn(navigator, 'userAgent', 'get').mockImplementation(() => 'Android');
+
+      let events = [];
+      let addEvent = (e) => events.push(e);
+      let res = render(
+        <Example
+          onPressStart={addEvent}
+          onPressEnd={addEvent}
+          onPress={addEvent}
+          onPressUp={addEvent} />
+      );
+
+      let el = res.getByText('test');
+      fireEvent(el, pointerEvent('pointerdown', {pointerId: 1, pointerType: 'mouse', width: 0, height: 0}));
+      fireEvent(el, pointerEvent('pointerup', {pointerId: 1, pointerType: 'mouse', width: 0, height: 0, clientX: 0, clientY: 0}));
+
+      expect(events).not.toEqual([]);
+
+      fireEvent.click(el);
+      expect(events).toEqual([
```

Comment:
shouldn't the second expect here be in place of the first expect? Don't think you need to test the `.click` here either since this is just making sure that Android pointer events with `width: 0` and `height: 0` aren't incorrectly classified as virtual events